open_fdd/gateway/local_codex_cli.py

```
@@ -0,0 +1,413 @@
+                errors="replace",
+            )
+        except (OSError, subprocess.TimeoutExpired):
+            continue
+        if r.stdout:
+            lines.extend([ln.strip() for ln in r.stdout.splitlines() if ln.strip()])
+    return lines
+
+
+def run_codex_login_status(codex_path: str) -> dict[str, Any]:
+    try:
+        cp = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)